notebooks/data-sources/TestGrid/metrics/time_to_fail.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>How do we interpret these plots? The first one makes sense, but how are the next two constructed, I guess the question is what are the parameters for invgauss.pdf() function?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>We are planning to add/discuss these parameters as a next step for this notebook - https://github.com/aicoe-aiops/ocp-ci-analysis/issues/336</p><p>We can add a description here as well :)</p>